Sources/BraveVPN/BraveVPN.swift

```diff
@@ -348,11 +379,42 @@ public class BraveVPN {
   }
   
   public static func populateRegionDataIfNecessary () {
-    serverManager.getRegionsWithCompletion { regions in
-      self.regions = regions
+    serverManager.regions { regions, _ in
+      self.regions = regions ?? []
+    }
+  }
+  
+  // MARK: - VPN Transport Protocol Functionality
```

Comment:
I would remove all those proxy methods, there's slim chance we'll make it configurable ever